bedrock/products/views.py

```diff
@@ -96,3 +105,135 @@ def vpn_invite_waitlist(request):
         resp = {"success": True}
 
     return JsonResponse(resp)
+
+
+def _build_category_list(entry_list):
```

Comment:
I don't see anything that would avoid dupes here. I could be missing something, but it looks possible for this to return multiple entries in the list for the same category.

Reply:
- [ ] Ensure we avoid duplicates @stevejalim

Reply:
Will split this into a separate ticket https://github.com/mozilla/bedrock/issues/10744